Sources/WebDriver/Keys.swift

```diff
@@ -1,11 +1,23 @@
 /// Represents a sequence of WebDriver key events and characters.
 public struct Keys: RawRepresentable {
+    /// A string encoding the key sequence as defined by the WebDriver spec.
     public var rawValue: String
 
     public init(rawValue: String) { self.rawValue = rawValue }
 
-    public static func +(lhs: Self, rhs: Self) -> Self { Self(rawValue: lhs.rawValue + rhs.rawValue) }
+    /// Concatenates multiple key sequences into a single one.
+    public static func sequence(_ keys: [Self]) -> Self {
+        Self(rawValue: keys.map(\.rawValue).joined())
```

Comment:
[flatMap](https://developer.apple.com/documentation/swift/sequence/flatmap(_:)-jo2y)?

Reply:
Why? 🤔 There are no arrays of arrays. I would understand suggesting [reduce](https://developer.apple.com/documentation/swift/sequence/reduce) though